store/sqlstore/channel_member_history_store.go

```diff
@@ -46,15 +49,18 @@ func (s SqlChannelMemberHistoryStore) LogJoinEvent(userId string, channelId stri
 }
 
 func (s SqlChannelMemberHistoryStore) LogLeaveEvent(userId string, channelId string, leaveTime int64) error {
-	query := `
-		UPDATE ChannelMemberHistory
-		SET LeaveTime = :LeaveTime
-		WHERE UserId = :UserId
-		AND ChannelId = :ChannelId
-		AND LeaveTime IS NULL`
-
-	params := map[string]interface{}{"UserId": userId, "ChannelId": channelId, "LeaveTime": leaveTime}
-	sqlResult, err := s.GetMaster().Exec(query, params)
+	query, params, err := s.getQueryBuilder().
+		Update("ChannelMemberHistory").
+		Set("LeaveTime", leaveTime).
+		Where(sq.And{
+			sq.Eq{"UserId": userId},
+			sq.Eq{"ChannelId": channelId},
+			sq.Eq{"LeaveTime": nil},
+		}).ToSql()
+	if err != nil {
```

Comment:
Let us be consistent throughout the codebase in how we handle errors from `ToSql`. The rest of the store wraps them in a standard convention using `errors.Wrapf(err, "<table>_tosql")`. Let's do that here too.

I see that in some places you are wrapping the error, and some places returning it directly.

Reply:
Hi @agnivade didn't know that. 

In some places I was returning the error by wrapping it because the function wrapped the rest of the errors. On the other places I didn't wrap it because the function returned the rest of the errors directly.

I'll change it through the weekend, I agree in being consistent!